### PR TITLE
Feat: Adding Cart Management page

### PR DESCRIPTION
**Pull request by arulk8**
1. Added Cart Management page
2. Fixed issue in product list page
3. Updated images in Home page

Site:
https://jolly-kringle-f186c8.netlify.app/

Pending: 
Wishlist
programmatically applying filter during navigation
mobile view

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import Cart from "./components/pages/Cart/Cart";
 import Wishlist from "./components/pages/Whishlist/Wishlist";
 import Login from "./components/pages/Authentication/Login";
 import SignUp from "./components/pages/Authentication/SignUp";
```
Comment (shraddha15-creator) [inline src/App.js:12]: use index.js file for import export

**Review by shraddha15-creator — APPROVED** [review]
Code is clean and well written
Easy to understand

**Comment by arulk8** [comment]
> Code is clean and well written Easy to understand

Thanks

File: src/components/Common/Components/CartCard/CartCard.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    discount,
+    img,
+    qty,
+    updateCartItem,
+    deleteFromCart,
+    addToWishlist,
+    wishlisted,
+    removeFromWishlist,
+  } = props;
+  const image = require(`../../../../assets/images/${img}.png`).default;
```
Comment (sanket4120) [inline src/components/Common/Components/CartCard/CartCard.jsx:18]: can put assets in public folder and you can access with a single / (e.g '/assets/imageName' )

**Review by sanket4120 — APPROVED** [review]
Rest looks good

**Pull request merged**